rules-tests/CodeQuality/Rector/Class_/InlineConstructorDefaultToPropertyRector/Fixture/skip_complex_assign.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
this `.php` fixture file seems commited during cut in the middle run unit test, please revert